.github/workflows/semantic-pr.yaml

```diff
@@ -0,0 +1,22 @@
+name: "Semantic Pull Request"
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
```

Comment:
Why not reopened as the [default configuration](https://github.com/amannn/action-semantic-pull-request?tab=readme-ov-file#installation) suggests?